### PR TITLE
fix: remove 3 launched feature flags

### PR DESCRIPTION
**Pull request by just-toby**
## Description

these 3 flags are true 100% of the time. removing them and the associated boilerplate and dead code.

_Slack thread:_ https://uniswapteam.slack.com/archives/C0574KNS77S/p1700001007805419

### Automated testing

- [x] Unit tests pass
- [x] Integration/E2E tests pass

### CODE REVIEW

File: src/state/routing/slice.ts [hunk 1]
```diff
@@ -71,15 +60,11 @@ function getRoutingAPIConfig(args: GetQuoteArgs): RoutingConfig {
     recipient: account,
   }
 
-  const tokenOutIsNative = Object.values(SwapRouterNativeAssets).includes(tokenOutAddress as SwapRouterNativeAssets)
-
   // UniswapX doesn't support native out, exact-out, or non-mainnet trades (yet),
```
Comment (tinaszheng) [inline src/state/routing/slice.ts:63]: update comment?

**Review by tinaszheng — APPROVED** [review]

File: src/components/Logo/AppleLogo.tsx [hunk 2]
```diff
@@ -1,8 +0,0 @@
-export const AppleLogo = (props: React.SVGProps<SVGSVGElement>) => (
```
Comment (kristiehuang) [inline src/components/Logo/AppleLogo.tsx:1]: nit can you also delete assets/svg/apple_logo.svg while you're at it?

**Review by kristiehuang — APPROVED** [review]
Lgtm with one nit to delete another apple logo svg we have - ty!! I love cleaning up

**Review by kristiehuang — APPROVED** [review]
yuh

**Pull request merged**